src/features/commentsSlice.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Avoid circular dependencies. If multiple slices need to share data, consider using a separate slice or custom middleware.